### PR TITLE
Enable PSA on 1.26 providers

### PR DESCRIPTION
**Pull request by acardace**
Signed-off-by: Antonio Cardace <acardace@redhat.com>

### CODE REVIEW

**Review by dhiller — APPROVED** [review]
/approve

/hold in case you want to bring in the psa check

File: cluster-provision/k8s/1.26/manifests/kubeadm.conf [hunk 1]
```diff
@@ -19,7 +19,11 @@ apiServer:
     audit-policy-file: /etc/kubernetes/audit/adv-audit.yaml
     enable-admission-plugins: NamespaceLifecycle,LimitRanger,ServiceAccount,DefaultStorageClass,DefaultTolerationSeconds,NodeRestriction,MutatingAdmissionWebhook,ValidatingAdmissionWebhook,ResourceQuota
     feature-gates: AdvancedAuditing=true
+    admission-control-config-file: /etc/kubernetes/psa.yaml
```
Comment (oshoval) [inline cluster-provision/k8s/1.26/manifests/kubeadm.conf:22]: Note please that there is a file named kubeadm_ipv6.conf
that should have the same changes

Comment (acardace) [reply cluster-provision/k8s/1.26/manifests/kubeadm.conf:22]: Done, thank you for catching this.

**Comment by oshoval** [comment]
Thanks

/lgtm

**Comment by dhiller** [comment]
/unhold

All tests succeeded

**Pull request merged**